mava/configs/env/matrax.yaml

```diff
@@ -25,4 +25,5 @@ scenario:
 eval_metric: episode_return
 
 kwargs:
-  time_limit: 500
+  # This value is in keeping with 'Benchmarking Multi-Agent Deep Reinforcement Learning Algorithms in Cooperative Tasks'
```

Comment:
Can you please remove this line, we follow the same idea for other env (e.g lbf)🙏

Reply:
Sure thing!